reference/media-types.md

```diff
@@ -213,6 +214,16 @@ its top level name. All of its sub-attributes will follow. For example, the
 above `default` view includes the `locale` attribute, which will cause its
 `language` and `country` sub-attributes to be rendered.
 
+A view will not render values that are `nil` by default. This can be overrided by passing `include_nil: true` when defining the view. For example:
```

Comment:
first sentence change=>  By default, a view will not render attributes whose values are `nil`.